docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.9"
+    
+services:
+  waltz:
+    image: ljubon/waltz:postgresql
+    restart: always
+    container_name: waltz
+    hostname: waltz
+    command: ["update","run"]
+    ports:
+      - "8080:8080"
+    depends_on:
+      - postgres
+    links:
+      - postgres
+    environment:
+      DB_HOST: postgres
+      DB_PORT: 5432
+      DB_NAME: waltz
+      DB_USER: waltz
+      DB_PASSWORD: waltz
+      DB_SCHEME: waltz
+      WALTZ_FROM_EMAIL: "ljubonikolic@gmail.com"
+      WALTZ_BASE_URL: "http://127.0.0.1:8080/"
+      CHANGELOG_FILE: "/liquibase/changelog/db.changelog-master.xml"
+    volumes:
+      - ./waltz-data/src/main/ddl/liquibase/:/liquibase/changelog
```

Comment:
I believe the changelog files should indeed not be mounted, but instead copied in the image and used from the image. Otherwise the user will need to hunt for the right changelog files for a given version of the Waltz image.